mono/utils/mono-context.c

```diff
@@ -418,6 +431,15 @@ mono_monoctx_to_sigctx (MonoContext *mctx, void *ctx)
 {
 #ifdef MONO_CROSS_COMPILE
 	g_assert_not_reached ();
+#elif defined(HOST_WIN32)
+	CONTEXT *context = (CONTEXT*)ctx;
+
+	context->Pc = mctx->pc;
+	context->Cpsr = mctx->cpsr;
+	memcpy (&context->R0, &mctx->regs, sizeof (DWORD) * 16);
+	
+	/* Why are we only copying 16 registers?! There are 32! */
```

Comment:
Copy/paste typo from previous line? Can you use sizeof(mctx->regs) and sizeof mctx->frgs) and C_ASSERT that they the same size as ->R0 and ->D? Is this actually 16 x 128bits?

Reply:
16 * 32 bits. DWORD is 4 bytes.

Reply:
Do we need to handle ymm and zmm? Do we need to dynamically query what the full context available is and use that?

Reply:
I'm not sure - apparently not according to this discussion: https://xamarinhq.slack.com/archives/C27F26XT4/p1519333216000258

Reply:
I think the code here is correct as-is. Although should we take another look?

Reply:
@joshpeterson it's all good :+1: